.github/scripts/toolchain/build-gcc.sh

```diff
@@ -59,6 +59,11 @@ if [[ "$RUN_CONFIG" = 1 ]] || [[ ! -f "$GCC_BUILD_PATH/Makefile" ]]; then
                     --without-libintl-prefix"
                 ;;
             *mingw*)
+                # ADDED: --enable-plugin
+                # REMOVED: --with-gmp=$TOOLCHAIN_PATH
```

Comment:
The `REMOVED` comments makes explicit what is the difference between this configuration and the upstream MSYS2 package configuration.